helm_drydock/drivers/node/maasdriver/api_client.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
```

Comment:
With their new library for interacting with maas, is this local copy and paste required?  I say this with the full knowledge that this approach used to be required.